### ISSUE TITLE
[Module 4: Images] Shall we use "Data Visualization"?

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-curricula#492]
The term "data viz" is widely used and understood. Integrating accessibility thinking into current hot topics is useful and I think the term should be used here when we use phrases like "images (including diagrams, charts, maps, and infographics) and their corresponding text alternatives..." and "alternatives for charts, diagrams, maps, and other complex graphics." We are often at risk of seeming out of date and using current terminology can help mediate that perception.

**Comment by daniel-montalvo** [comment]
Hey @sharronrush 

That is very good suggestion.

I will take a pass at it.

Maybe we could have:

* module 3: text Alternatives -- including text alternatives for functional and informative images mainly
* Module 4: Data Visualization -- including text alternatives for complex images, charts, infographics, and others


Will update this issue as I progress on this.

**Comment by daniel-montalvo** [comment]
First attempt to include the concept of "Data Visualizations".

Currently:

- [Module 3: Text Alternatives](https://content-author-topics--wai-curricula.netlify.app/curricula/content-author-modules/text-alternatives/) -- focuses on text alternatives for functional and informative images.
- [Module 4: Data Visualizations](https://content-author-topics--wai-curricula.netlify.app/curricula/content-author-modules/data-visualizations) -- focuses on different types of data visualizations, including complex images and data tables.

**Comment by daniel-montalvo** [comment]
This separation seemed awkward for the TF in [yesterday's discussion](https://www.w3.org/2022/03/22-wai-curricula-minutes.html).

The TF appreciates consistency with the developers and designers curricula, where we tend to have module titles around specific pieces of content.

Will go back to "Images" and "Tables" and will try to include these mentions to "Data visualizations" where appropriate in these two.

**Comment by daniel-montalvo** [comment]
Addressed as per [EOWG Meeting 25 March](https://www.w3.org/2022/03/25-eo-minutes.html#t02)
